modules/openapi-generator/src/test/resources/3_0/python/petstore-with-fake-endpoints-models-for-testing.yaml

```
@@ -1532,6 +1532,67 @@ components:
           type: string
       xml:
         name: Tag
+    PoopCleaning:
+      type: object
+      properties:
+        task_name:
+          type: string
+          enum: [cleaning]
+        function_name:
+          type: string
+          enum: [care]
+        content:
+          type: string
+      required:
+        - task_name
+        - function_name
+        - content
+    Feeding:
+      type: object
+      properties:
+        task_name:
+          type: string
+          enum: [cleaning]
```

Nitpicking, but I think this should be `feeding` instead :) (or something similar)
```suggestion
          enum: [feeding]
```